### ISSUE TITLE
Custom Thumbnail for Youtube, Vimeo,.. embedded Videos

### ISSUE THREAD

**Issue by masteradhoc** [WordPress/gutenberg#46976]
## What problem does this address?
Sometimes the Thumbnail of the video that gets embedded shouldnt be automatically delivered by the video hoster - it should be manually set. This is currently not possible within the editor.

## What is your proposed solution?
Allow us to specifically set the thumbnail in the block settings.

**Comment by Mamaduka** [comment]
The editor just uses data from oEmbed providers to display previews; the resource itself is stored as links, and WP automatically embeds the content on the frontend.

So embedded content, including a thumbnail, is automatically delivered by the video hoster and WP makes no modifications to it. This system pre-dates the block editor, and there haven't been many changes to how it works.

Since the editor can't modify the embedded content, I'm going to close this as "can't fix".

**Comment by masteradhoc** [comment]
Seems its possible. For those that need a fix, here you go: https://wordpress.org/plugins/better-core-video-embeds/

**Comment by Mamaduka** [comment]
Yes, the plugin can make more substantial modifications, like [Lazy Embeds](https://wordpress.org/plugins/lazy-embeds/). Core just displays what the provider returns, making it more predictable for plugins to modify.

There's also YouTube's terms of service, which prohibits some modifications of embedded content - https://developers.google.com/youtube/terms/required-minimum-functionality?hl=en#overlays-and-frames

> You must not display overlays, frames, or other visual elements in front of any part of a YouTube embedded player, including player controls. Similarly, you must not use overlays, frames or other visual elements to obscure any part of an embedded player, including player controls.
